CHANGES.md

```diff
@@ -15,7 +15,23 @@
 [Unreleased](https://github.com/bird-house/birdhouse-deploy/tree/master) (latest)
 ------------------------------------------------------------------------------------------------------------------
 
-[//]: # (list changes here, using '-' for each new entry, remove this when items are added)
+# Changes
+
+- Update Docker Compose syntax to version 2 in all docker compose files
+
+  The version 1 compose specification that docker uses is out of date and no longer maintained by docker.
+  We currently have a mix of version 1 syntax and version 2 (specifically 2.20.2+) syntax in our docker compose
+  files.
+
+  This means that the stack will not properly run with a docker compose version <2.20.2. For any version >2.20.2 
+  the stack runs properly but displays lots of deprecation warnings about deprecated version strings and external
```

Comment:
Here, the `version` should be highlighted since this message out of context can be ambiguous. 
i.e.: link to https://docs.docker.com/reference/compose-file/version-and-name/